src/init/command.template

```diff
@@ -7,5 +7,26 @@
 cd { host_shared }
 {{ endif }}
 
+# Discover where the output chardev is located
+vport=
+for dir in /sys/class/virtio-ports/*; do
+    if [[ "$(cat "$dir/name")" == "{ command_output_port_name }" ]]; then
+        vport_name=$(basename "$dir")
+        vport="/dev/$vport_name"
+    fi
+done
+
+# Send the rest of the script to the output chardev
+if [[ -n "$vport" ]]; then
+    exec > "$vport"
+    exec 2>&1
```

Comment:
It may be valuable in the future to have stdout and stderr separated, but this is a good trade-off for now.